sdk/kusto/Azure.ResourceManager.Kusto/src/Generated/Models/FollowerDatabaseDefinitionGet.cs

```diff
@@ -0,0 +1,81 @@
+        /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="FollowerDatabaseDefinitionGet"/>. </summary>
+        internal FollowerDatabaseDefinitionGet()
+        {
+        }
+
+        /// <summary> Initializes a new instance of <see cref="FollowerDatabaseDefinitionGet"/>. </summary>
+        /// <param name="clusterResourceId"> Resource id of the cluster that follows a database owned by this cluster. </param>
+        /// <param name="attachedDatabaseConfigurationName"> Resource name of the attached database configuration in the follower cluster. </param>
+        /// <param name="databaseName"> The database name owned by this cluster that was followed. * in case following all databases. </param>
+        /// <param name="tableLevelSharingProperties"> Table level sharing specifications. </param>
+        /// <param name="databaseShareOrigin"> The origin of the following setup. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal FollowerDatabaseDefinitionGet(string clusterResourceId, string attachedDatabaseConfigurationName, string databaseName, KustoDatabaseTableLevelSharingProperties tableLevelSharingProperties, KustoDatabaseShareOrigin? databaseShareOrigin, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            ClusterResourceId = clusterResourceId;
+            AttachedDatabaseConfigurationName = attachedDatabaseConfigurationName;
+            DatabaseName = databaseName;
+            TableLevelSharingProperties = tableLevelSharingProperties;
+            DatabaseShareOrigin = databaseShareOrigin;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Resource id of the cluster that follows a database owned by this cluster. </summary>
+        public string ClusterResourceId { get; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/maps/Azure.ResourceManager.Maps/src/autorest.md?plain=1#L25).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://github.com/Azure/azure-sdk-for-net/blob/bd953fc7a97bf617c892ce6314b50126b4021a4f/sdk/search/Azure.Search.Documents/src/autorest.md?plain=1#L106-L115).

Reply:
changed to ResourceIdentifier